lib/isMobilePhone.js

```diff
@@ -23,6 +23,7 @@ var phones = {
   'ar-SY': /^(!?(\+?963)|0)?9\d{8}$/,
   'ar-TN': /^(\+?216)?[2459]\d{7}$/,
   'be-BY': /^(\+?375)?(24|25|29|33|44)\d{7}$/,
+  'bn-BN': /\+?(88)?0?1[156789][0-9]{8}\b/,
```

Comment:
I'm not really sure if `-BN` is the code for Bangladesh given that it's ccTLD is `.bd`, [see here](https://en.wikipedia.org/wiki/.bd), while [`.bn` is for Brunei](https://en.wikipedia.org/wiki/.bd).

Where can we get some reference for this?

Reply:
Quick search, I have seen [`bn-rBD` being used here](https://github.com/championswimmer/android-locales) for instance.

Reply:
It'll be 'bn-BD', I'll fix that.